util/site/build-docs.sh

```diff
@@ -157,10 +157,12 @@ buildSite () {
     mkdir -p "${rustdoc_dir}"
     local bazel_out target_rustdoc target_rustdoc_output_path
     bazel_out="$(./bazelisk.sh info output_path 2>/dev/null)"
-    target_rustdoc="sw/host/opentitanlib:opentitanlib_doc"
-    target_rustdoc_output_path="${bazel_out}/k8-fastbuild/bin/$(echo ${target_rustdoc} | tr ':' '/').rustdoc" #TODO : get the target's path using cquery
-    ./bazelisk.sh build --experimental_convenience_symlinks=ignore "${target_rustdoc}"
-    cp -rf "${target_rustdoc_output_path}"/* "${rustdoc_dir}"
+    for target_rustdoc in "sw/host/opentitanlib:opentitanlib_doc" "sw/host/hsmtool:hsmlib_doc" "sw/host/ot_certs:ot_certs_doc"
+    do
+      target_rustdoc_output_path="${bazel_out}/k8-fastbuild/bin/$(echo ${target_rustdoc} | tr ':' '/').rustdoc" #TODO : get the target's path using cquery
+      ./bazelisk.sh build --experimental_convenience_symlinks=ignore "${target_rustdoc}"
+      cp -rf "${target_rustdoc_output_path}"/* "${rustdoc_dir}"
+    done
```

Comment:
This should fix the failing build in CI. It fails for me locally too in the same way.
```suggestion
      cp -rf "${target_rustdoc_output_path}"/* "${rustdoc_dir}"
      chown -R $USER: "${rustdoc_dir}"
      chmod -R +w "${rustdoc_dir}"
    done
```